src/Agent/NewRelic/Agent/Core/WireModels/LoadedModuleWireModelCollection.cs

```diff
@@ -71,11 +70,17 @@ public static LoadedModuleWireModelCollection Build(IList<Assembly> assemblies)
 
         private static bool TryGetAssemblyName(Assembly assembly, out string assemblyName)
         {
+            // We skip any assemblies that cannot be linked to a nuget package.
+
             try
             {
-                if (assembly.IsDynamic)
+                if (assembly.IsDynamic) // skip dynamic assemblies
+                {
+                    assemblyName = null;
+                }
+                else if (assembly.GetName().Version.ToString() == "0.0.0.0") // skip assemblies that have a version of 0.0.0.0 - these are precompiled assemblies
```

Comment:
Since we could be checking many thousands of these, it might be faster to use the `==` operator on the `Version`  rather than a string conversion?

Reply:
Yeah, your right. I will fix it up.